src/client/api.js

```diff
@@ -1,8 +1,12 @@
-function foo1(a, b, a) {
+function foo1(a, b, c) {
     typeof a === undefined
     console.log("value of the second a:", a);
 }
 
-var bar = function (a, b, a) {
+var bar = function (a, b, c) {
     console.log("value of the second a:", a);
-};
\ No newline at end of file
+};
+
+var car = function (a, b, c) {
+    console.log("value of the second c:", c);
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](http://ec2-50-19-3-36.compute-1.amazonaws.com/gh/codacy-acme/nodeproject/pullRequest?prid=1)
```suggestion
    console.log('value of the second c:', c);
```